Prox/Prox/Data/TravelTimesProvider.swift

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import MapKit
+
+struct TravelTimesProvider {
+
+    static func travelTime(fromLocation: CLLocationCoordinate2D, toLocation: CLLocationCoordinate2D, byTransitType transitType: MKDirectionsTransportType = .any, withCompletion completion: @escaping ((TravelTimes?) -> ())) {
+
+        let directionsRequest = MKDirectionsRequest()
+        if #available(iOS 10.0, *) {
+            directionsRequest.source = MKMapItem(placemark: MKPlacemark(coordinate: fromLocation))
+            directionsRequest.destination = MKMapItem(placemark: MKPlacemark(coordinate: toLocation))
+        } else {
+            directionsRequest.source = MKMapItem(placemark: MKPlacemark(coordinate: fromLocation, addressDictionary: nil))
```

Comment:
the only advantage is if they decide to deprecate the older one at a future date. As that is a future date rather than immediate, I'll make the change you suggest